tests/test_utils.py

```diff
@@ -189,3 +191,37 @@ def test_intensity_image(ndim):
     for obj in objects:
         centroid = (int(obj.z), int(obj.y), int(obj.x))[-ndim:]
         assert obj.properties["max_intensity"] == intensity_image[centroid]
+
+
+def _load_segmentation_and_tracks():
+    f = h5py.File("./tests/_test_data/update_segmentation_data.h5", "r")
+    coords = tuple(f[c][:] for c in ["tc", "yc", "xc"])
+    in_segmentation = np.zeros((10, 1024, 1020), dtype=f["in_values"].dtype)
+    out_segmentation = np.zeros((10, 1024, 1020), dtype=f["out_values"].dtype)
+    in_segmentation[coords] = f["in_values"][:]
+    out_segmentation[coords] = f["out_values"][:]
+    tracks = HDF5FileHandler("./tests/_test_data/tracks.h5").tracks
```

Comment:
Do you need to close `f`? Likewise, I would recommend using `HDF5FileHandler` with the context manager, as it takes care of closing the H5 file properly:

```python
with HDF5FileHandler("./tests/_test_data/tracks.h5", "r") as hdf:
    tracks = hdf.tracks
```

This shouldn't matter here, but is good practice